basket/news/utils.py

```diff
@@ -280,9 +280,10 @@ def language_code_is_valid(code):
 ]
 
 
-def get_user_data(token=None, email=None, extra_fields=None, get_fxa=False):
+def get_user_data(token=None, email=None, stripe_id=None, extra_fields=None, get_fxa=False):
```

Comment:
Still working on making this work I guess?

Reply:
Yeah - just put this in as a starting point.